packages/core/src/components/forms/_control-group.scss

```diff
@@ -126,13 +128,10 @@ Styleguide components.forms.control-group
     position: relative;
   }
 
-  .pt-input-group .pt-button:focus {
```

Comment:
Clueless about what this rule is for. What if we just remove it instead of introducing `.pt-input-action`?